COMPILING.md

```diff
@@ -46,15 +46,15 @@ $ git clone https://github.com/google/protobuf.git
 $ cd protobuf
 $ git checkout v3.7.0
 $ ./autogen.sh
-$ ./configure
+$ ./configure --disable-shared
 $ make
 $ make check
 $ sudo make install
 ```
 
 If you are comfortable with C++ compilation and autotools, you can specify a
 ``--prefix`` for Protobuf and use ``-I`` in ``CXXFLAGS``, ``-L`` in
-``LDFLAGS``, ``LD_LIBRARY_PATH``, and ``PATH`` to reference it. The
+``LDFLAGS`` to reference it. The
 environment variables will be used when building grpc-java.
```

Comment:
Ideally this line would have been reflowed and the comma would have been replaced with "and". But whatever.